### DISCUSSION TITLE
A metadata API route for the back end

### DISCUSSION THREAD

**Discussion by KairuDeibisu** [clerk/javascript#1966] [Feature Requests]
## Package

- [x] `@clerk/clerk-react`
- [x] `@clerk/backend`

My site will send student's text messages, if the tutor creates a lesson.

When a text message needs to be sent, I need to get the users metadata.

I don't need the entire user object just the metadata to get the notification preferences.

Exposing an API route that grabs all three types of metadata specifically, given the user id, It would be quite useful. Currently we can only update the metadata, and not get it specifically.

**Comment by LekoArts** [comment]
Hello!

We're tracking our feature requests now in [feedback.clerk.com](https://feedback.clerk.com). If your suggestion is already in the list of proposed changes, please upvote. Otherwise, please create an entry. Thanks!

Our feedback platform will be the source of truth for feature requests going forward and your input and votes will help us shape the roadmap.

Thanks for using Clerk! 💙
